src/main/java/org/zalando/nakadi/controller/TimelinesController.java

```diff
@@ -66,8 +68,10 @@ public TimelinesController(final TimelineService timelineService) {
         return Responses.create(MoreStatus.UNPROCESSABLE_ENTITY, ex.getMessage(), request);
     }
 
-    @ExceptionHandler(NotFoundException.class)
-    public ResponseEntity<Problem> notFound(final NotFoundException ex, final NativeWebRequest request) {
+    @ExceptionHandler({
+            NotFoundException.class,
+            TimelinesNotSupportedException.class})
```

Comment:
Wouldn't it make more sense to reply with 422? It actually found the event type but could not go on with the request.

Reply:
Oh yes, sorry. I just added it to wrong handler accidentally. Will fix.

Reply:
fixed https://github.com/zalando/nakadi/pull/907/commits/939733792a424c68b893a490349e03a6bf343b62